src/ert/config/queue_config.py

```diff
@@ -115,24 +119,41 @@ def max_running(self) -> int:
         return max_running
 
 
+def _option_list_to_dict(option_list: List[Tuple[str, str]]) -> Dict[str, List[str]]:
+    temp_dict: Dict[str, List[str]] = defaultdict(list)
+    for option_string in option_list:
+        temp_dict.setdefault(option_string[0], []).append(option_string[1])
+    return temp_dict
+
+
 def _check_for_overwritten_queue_system_options(
     selected_queue_system: QueueSystem,
     queue_system_options: List[Tuple[str, str]],
 ) -> None:
-    def generate_dict(option_list: List[Tuple[str, str]]) -> Dict[str, List[str]]:
-        temp_dict: Dict[str, List[str]] = defaultdict(list)
-        for option_string in option_list:
-            temp_dict.setdefault(option_string[0], []).append(option_string[1])
-        return temp_dict
-
-    for option_name, option_values in generate_dict(queue_system_options).items():
+    for option_name, option_values in _option_list_to_dict(
+        queue_system_options
+    ).items():
         if len(option_values) > 1 and option_values[0] != option_values[-1]:
             logging.info(
                 f"Overwriting QUEUE_OPTION {selected_queue_system} {option_name}:"
                 f" \n Old value: {option_values[0]} \n New value: {option_values[-1]}"
             )
 
 
+def _check_num_cpu_requirement(
+    num_cpu: int,
+    queue_system_options: List[Tuple[str, str]],
+) -> None:
+    torque_options = _option_list_to_dict(queue_system_options)
+    num_nodes = int(torque_options.get("NUM_NODES", [1])[0])
+    num_cpus_per_node = int(torque_options.get("NUM_CPUS_PER_NODE", [1])[0])
+    if num_cpu > num_nodes * num_cpus_per_node:
+        raise ConfigValidationError(
```

Comment:
I don't get this tbh. If user specifies `NUM_CPU` and PBS (with `NUM_CPUS_PER_NODE` and `NUM_NODES`) then `PBS` takes precedence or is not?

Reply:
Setting to equality.